appstudio-controller/Makefile

```diff
@@ -258,6 +258,7 @@ endef
 lint:
 	golangci-lint --version
 	GOMAXPROCS=2 golangci-lint run --fix --verbose --timeout 300s
+	golangci-lint run -E ginkgolinter ./...
```

Comment:
This will run all the linter again, in addition to ginkgolinyer. What about adding the ginkgolinter to the .golanci.yml file?

Reply:
yes, adding ginkgolinter to the .golanci.yml file sound good, I will add it

Reply:
Hey @nunnatsa :wave:, thanks!